cmd/sys-mgmt-agent/res/configuration.toml

```diff
@@ -57,3 +57,19 @@ Type = 'consul'
   Protocol = 'http'
   Host = 'localhost'
   Port = 48085
+
+[SecretStore]
+Type = 'vault'
+Protocol = 'http'
+Host = 'localhost'
+Port = 8200
+Path = '/v1/secret/edgex/sys-mgmt-agent/'
+TokenFile = '/vault/config/assets/resp-init.json'
```

Comment:
Note that for App Services I have been using the `/tmp/edgex/secrets/... ` location of the token that was created from Secret Store running in docker. I assume this is also overridden in Snap for its specific location Correct?

Something we should consider doing for all the services, but is out of scope for this PR.

Reply:
yes, `system-agent` has its overriden in snap as well.

Reply:
the majority of core- services follow the current way as I did above. not sure which way to standardize in future. maybe worth a discussion in next Weekly Security WG or Core WG meeting.  
@jpwhitemn  and @hutchic, thoughts?

Reply:
Yup, we should discuss in next Security WG.

 Snap will always need the override since the root of the path is ` $SNAP_DATA`. 
IMO, we should standardize the default config on using `/tmp/edgex/secrets/...` so we can eliminate a bunch of env overrides and allows me to run the service from command-line and it will find the token generated from the docker stack.